packages/types/src/scripts/srmlTS.ts

```diff
@@ -22,6 +22,7 @@ interface TypeImports {
 
 const HEADER = '/* eslint-disable @typescript-eslint/no-empty-interface */\n// Auto-generated, do not edit\n\n';
 const FOOTER = '\n';
+const OUTPUT_FILE = 'index.ts';
```

Comment:
See initial PR comment - on the fence around the naming, `types.ts` probably fits better and it clearer in intent (although slightly more effort to use)